docs/source/bag-api.rst

```diff
@@ -0,0 +1,32 @@
+API
+===
+
+.. currentmodule:: dask.bag.core
+
+Create Bags
+-----------
+
+.. autosummary::
+   from_sequence
+   from_filenames
+   concat
+
+Turn Bags into other things
+---------------------------
+
+.. autosummary::
+   Bag.to_textfiles
+   Bag.to_dataframe
+
+Bag methods
+-----------
+
+.. autoclass:: Bag
+   :members:
```

Comment:
I think this is really helpful. I manually added a full autosummary at the top with the public members of Bag. But there [doesn't seem to be an automatic way to do this with autosummary](http://stackoverflow.com/questions/20569011/python-sphinx-autosummary-automated-listing-of-member-functions) other than extending it manually.